File: LeetCode/657.Robot_Return_Origin/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    string string1 = "UD";
    string string2 = "LL";
    
    if (s.judgeCircle(string1) == true) { cout << "Test 1 - PASS" << endl;}
    else { cout << "Test 1 - Failed" << endl; }
    if (s.judgeCircle(string2) == false) { cout << "Test 2 - PASS" << endl;}
    else { cout << "Test 2 - Failed" << endl; }
    return 0;
}

File: LeetCode/657.Robot_Return_Origin/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test
string1 = "UD"
string2 = "LL"
string3 = "LDRRLRUULR"
s = Solution()
if (s.judgeCircle(string1)==True):
    print("PASS")
else:
    print("TEST 1 FAILED")

if (s.judgeCircle(string2)==False):
    print("PASS")
else:
    print("TEST 2 FAILED")


if (s.judgeCircle(string3)==False):
    print("PASS")
else:
    print("TEST 2 FAILED")